ui/dashboards/src/components/SaveChangesConfirmationDialog/SaveChangesConfirmationDialog.tsx

```diff
@@ -0,0 +1,104 @@
+// Copyright 2023 The Perses Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { useState } from 'react';
+import {
+  IconButton,
+  Dialog,
+  DialogTitle,
+  DialogContent,
+  DialogActions,
+  Button,
+  Checkbox,
+  FormGroup,
+  FormControlLabel,
+  Typography,
+} from '@mui/material';
+import CloseIcon from 'mdi-material-ui/Close';
+import { useTimeRange } from '@perses-dev/plugin-system';
+import { isRelativeTimeRange, SAVE_DEFAULTS_DIALOG_TEXT } from '@perses-dev/core';
+import { useSaveChangesConfirmationDialog } from '../../context';
+
+export const SaveChangesConfirmationDialog = () => {
+  const [saveDefaultTimeRange, setSaveDefaultTimeRange] = useState(true);
+  const [saveDefaultVariables, setSaveDefaultVariables] = useState(true);
+
+  const { saveChangesConfirmationDialog: dialog } = useSaveChangesConfirmationDialog();
+  const isOpen = dialog !== undefined;
+
+  const { timeRange } = useTimeRange();
+  const currentTimeRangeText = isRelativeTimeRange(timeRange)
+    ? `(Last ${timeRange.pastDuration})`
+    : '(Absolute time ranges can not be saved)';
+
+  const timeRangeInfoText = `Save current time range as new default ${currentTimeRangeText}`;
+
+  return (
+    <Dialog open={isOpen}>
+      {dialog !== undefined && (
+        <>
+          <DialogTitle>Save Dashboard</DialogTitle>
```

Comment:
We actually have a [reusable `Dialog` component](https://github.com/perses/perses/tree/main/ui/components/src/Dialog) in perses so you don't need to add these custom styles anymore!

Reply:
Can't believe I forgot about that! I was being lazy and copied DiscardChangesConfirmationDialog.tsx into SaveChangesConfirmationDialog.tsx (and just changed the 'Discard' to 'Save'), might do a follow-up PR to backfill all dialogs to use the new one from @perses-dev./components